src/bric.c

```diff
@@ -1603,15 +1606,15 @@ int handle_tag_movement(int where) {
 			editor_move_cursor(ARROW_UP);
                 for(; i > 0; i--)
 			editor_move_cursor(ARROW_DOWN);
-                return 1;		
+                return 1;
 	}
 	if(key[0] == '\0') {
-		editor_set_status_message("No identifier under cursor");
+		editor_set_status_message(_("No identifier under cursor"));
```

Comment:
"identifier" may be ambiguous. I suggest leaving a TRANSLATOR note describing what an identifier specifically refers to.